pyproject.toml

```diff
@@ -31,6 +31,7 @@ dependencies = [
     "biocommons.seqrepo",
     "gene-normalizer ==0.4.0",
     "cool-seq-tool ~=0.11.0",
+    "civicpy ==4.1.0"
```

Comment:
I'm not sure we want to pin a specific version for CIViCPy. Consider only pinning the major version.

The reason why we pinned exact versions for normalizers/etc is because the python reference implementations where in an alpha state and could introduce breaking changes in the next alpha version. However, now that we're back in a stable version we can start to remove the exact pins.